sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/aio/azure_configuration_client_async.py

```diff
@@ -68,17 +90,19 @@ def _create_appconfig_pipeline(self):
             AppConfigRequestsCredentialsPolicy(self.config.credentials),
             self.config.retry_policy,
```

Comment:
Does the default retry policy handle `x-ms-retry-after-ms` headers in responses?

Reply:
> x-ms-retry-after-ms

No. Default retry policy does not handle x-ms-retry-after-ms headers